docs/technical-details/reports/right-sizer.md

```diff
@@ -1,21 +1,109 @@
 # Right-sizer
-Right-sizer is a Kubernetes Controller that detects and optionally modifies containers that have run out of memory and been OOM-killed and maintains an Insights report of those containers with their owning pod-controllers (such as Deployments, StatefulSets or DaemonSets). The Controller persists report data in a Kubernetes ConfigMap which the accompanying Insights Agent CronJob retrieves and submits to the Insights API.
 
-The right-sizer controller removes pod-controllers from its report when no OOM-kill has been seen within a time window (default 24 hours) or if the memory limits have been modified compared to the limits from the last-seen OOM-kill. 
+As of Insights Agent version `4.0`, Right-sizer now refers to multiple capabilities:
+
+* A controller that manages VPA objects for automated right-sizing, and
+* A controller that detects and modifies contains that have been OOM-killed
+
+These are currently separate binaries and container images:
+
+https://quay.io/repository/fairwinds/insights-right-sizer: refers to the VPA controller
+
+https://quay.io/repository/fairwinds/right-sizer: refers to OOM-kill detection and resolution
+
+The intention is to consolidate both of these in a future release once more testing has been done.
+
+## Breaking Changes
+
+ Version 4.0
+> The 4.0 release of insights-agent contains breaking changes to `right-sizer`. This component has been rebranded to refer to Insights automated right sizing. The `right-sizer` prior to this release will be referred to as `oom-detection` going forward. These will be further consolidated in a future release to avoid confusion.
+
+## Automated Workload Right-sizing
 
 ## Setup and Configuration
-To quickly get going with the right-sizer you can enable it by adding the following to your `values.yaml` file.
 
-> If you're using a version of Insights Agent below 2.0 you will need to remove the hyphen between right-sizer from the below value.
+Right-sizer includes a custom `recommender` VPA component, we've modified the recommender to retrieve recommendations from Insights. When `right-sizer` is enabled, we will install the [Fairwinds VPA](https://artifacthub.io/packages/helm/fairwinds-stable/vpa) helm chart with this custom image for you.
```

Comment:
There are a bunch of implementation details here that I think we can hide from the user, or push down into a n "Implementation Details" section

I'd start this with a sample `values.yaml` that can be passed to the helm chart (i.e. instead of the ConfigMap below)